packages/react/src/index.ts

```diff
@@ -14,3 +14,4 @@ export * from './FAQ'
 export * from './forms'
 export * from './Stack'
 export * from './Testimonial'
+export * from './DevGrid'
```

Comment:
Crazy idea, but what do you think about publishing this as a separate package? Something like `@primer/brand-devtools`.. it would still live in this repo, just isolated and we can keep it enabled in Storybook as a dependency.

Reply:
I like that idea